docs/entities/serviceclass.md

```diff
@@ -1,20 +1,58 @@
 # ServiceClass
 
+A Service Class is intended to define how a registered service can be
+automatically generated from a service.  Currently, this service takes the
+shape of a resource already existing within a cluster; the Service Class
+controller will recognize those services and create a corresponding registered
+service.
```

Comment:
For a better review experience please format the markdown as follow. One line for each sentence, one empty line for creating a new paragraph.

```suggestion
A Service Class is intended to define how a registered service can be automatically generated from a service.
Currently, this service takes the shape of a resource already existing within a cluster;
the Service Class controller will recognize those services and create a corresponding Registered Service.
```

Reply:
Fixed.